recipe/meta.yaml

```diff
@@ -129,15 +124,15 @@ requirements:
   - krb5==1.14.2
   - libpng>=1.6.34
   - libprotobuf==3.5.2
-  - libtiff=>4.0.9
+  - libtiff>=4.0.9
   - libxml2==2.9.7
   - mesalib==17.2.0
   - mysql==5.7.20
   - mysql-connector-c==6.1.6
   - nanoflann==1.2.2
   - conda-forge/label/gcc7::nn
   - openblas==0.2.19
-  - opencv==3.2.0
+  - opencv
```

Comment:
Opening up OpenCV can break a lot of things. We have this hard pinned to 3.2.0 so that we get the "non-free" packages and the correct interface for findfeatures. Why was this changed?

Reply:
The section was copied from environment.yml, which does not specify a version for opencv.

Reply:
I'll change environment back to including boost + explicitly versioning opencv, but should I also change the environment.yml file?  Or change the docs that state that these sections should be copied from environment.yml?  Things seem a little out of sync here.

Reply:
The two files are out of synce because of how channels are specified in the two. In an environment file we can specify a channel priority, in a conda-build meta.yml file we cannot. Having things more pinned in the meta.yaml helps ensure all of the correct packages are installed without the channel priority. For the 4.1 release, modifying the meta.yaml make sense as we are working on dependencies, but the 3.10 release doesn't have any of that in it, so we want to keep using the old environment.